packages/element-lib/src/adapters/database/ElementRXDBAdapter.js

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable no-underscore-dangle */
+const RxDB = require('rxdb');
+
+class ElementRXDBAdapter {
+  constructor({ name }) {
+    // RXDB name regex is quite restrictive, therefore we have to replace
+    // some special characters like "." and "-" with "_"
+    // and put all letters to lowercase
+    this.name = name.replace(/\.|-/g, '_').toLowerCase();
+    if (process.browser) {
+      RxDB.plugin(require('pouchdb-adapter-idb'));
+      this.adapter = 'idb';
+    } else {
+      RxDB.plugin(require('pouchdb-adapter-leveldb'));
+      this.adapter = require('memdown');
```

Comment:
This is in memory, I think in the long run, we want to use RxDB with CouchDB hosted on cloudant, or some other couchdb as a service offering. This is a good intermediate step though.

Reply:
Yes, and this is used only for the tests. The browser one will be used in prod